dockerfiles/packaging/debian/package/etc/james/jmap.properties

```diff
@@ -0,0 +1,13 @@
+# Configuration file for JMAP
+
+enabled=true
+
+tls.keystoreURL=file://conf/keystore
```

Comment:
No, to be consistent with the docker image.
Plus, James started without error after installing it.